ToyBox/classes/MonkeyPatchin/Multiclass/LevelUP+Multiclass.cs

```diff
@@ -140,10 +140,12 @@ private static class ApplySpellbook_Apply_Patch {
                         var spellsKnown = spellbook1.Blueprint.SpellsKnown;
                         var expectedCount = spellsKnown.GetCount(casterLevelAfter, index);
                         var actual = CasterHelpers.GetCachedSpellsKnown(unit, spellbook1, index);
+                        int learnabl = spellbook1.GetSpellsLearnableOfLevel(index).Count();
+                        int spelladd = Math.Max(0, Math.Min(expectedCount - actual, learnabl));
```

Comment:
I will note that it's still possible to screw this up by folks adding spells from other classes.